### ISSUE TITLE
Provide way to disable implicit table header

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#1960]
Now that setting the options attribute does not disable the check for an implicit table header, there needs to be a way to disable the implicit table header. Perhaps the option `noheader` will suffice.

```
[%noheader]
|===
|Row 1, Cell 1 | Row 1, Cell 2

|Row 2, Cell 1| Row 2, Cell 2
|===
```

**Comment by mojavelinux** [comment]
Note that one workaround is to put a line comment on the line after the first row.

**Comment by mojavelinux** [comment]
As it turns out, this was already implemented by #1849, so this is a duplicate.
